src/run.ts

```diff
@@ -173,20 +172,17 @@ export class KustomizeRenderEngine extends RenderEngine {
 
         core.debug("Running kubectl kustomize command..");
         console.log(`[command] ${kubectlPath} kustomize ${core.getInput('kustomizationPath')}`);
-        var result = await utilities.execCommand(kubectlPath, ['kustomize', kustomizationPath], options);
+        const result = await utilities.execCommand(kubectlPath, ['kustomize', kustomizationPath], options);
         const pathToBakedManifest = this.getTemplatePath();
         fs.writeFileSync(pathToBakedManifest, result.stdout);
         core.setOutput('manifestsBundle', pathToBakedManifest);
     };
 
     private async validateKustomize(kubectlPath: string) {
-        var result = await utilities.execCommand(kubectlPath, ['version', '--client=true', '-o', 'json']);
+        const result = await utilities.execCommand(kubectlPath, ['version', '--client=true', '-o', 'json']);
         if (!!result.stdout) {
             const clientVersion = JSON.parse(result.stdout).clientVersion;
-            if (clientVersion && parseInt(clientVersion.major) >= 1 && parseInt(clientVersion.minor) >= 14) {
-                // Do nothing
-            }
-            else {
+            if (!clientVersion || parseInt(clientVersion.major) < 1 || parseInt(clientVersion.minor) < 14) {
```

Comment:
i think this logic will throw error on higher major versions with minor<14 ex: 2.13

Reply:
maybe add unit tests for checking this to make sure?

Reply:
`!clientVersion || (clientVersion.major <=1 && clientVersion.minor < 14)`